rmw_opensplice_cpp/src/functions.cpp

```diff
@@ -6,7 +6,6 @@
 #include <rmw/allocators.h>
 #include <rmw/error_handling.h>
 #include <rmw/rmw.h>
-#include <rosidl_generator_c/message_type_support.h>
```

Comment:
Why remove this? It provides the symbol `rosidl_message_type_support_t` which is used in this file.